.github/workflows/pr_test_build_linux.yml

```diff
@@ -0,0 +1,186 @@
+name: PR Test Build linux
+
+on:
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Branch name to build"
+        required: true
+        default: "main"
+
+jobs:
+  PR_test_build:
+    runs-on: ubuntu-20.04
+    env:
+      STORE_PASS: test@cake_wallet
+      KEY_PASS: test@cake_wallet
+      PR_NUMBER: ${{ github.event.number }}
+
+    steps:
+      - name: is pr
+        if: github.event_name == 'pull_request'
+        run: echo "BRANCH_NAME=${GITHUB_HEAD_REF}" >> $GITHUB_ENV
+
+      - name: is not pr
+        if: github.event_name != 'pull_request'
+        run: echo "BRANCH_NAME=${{ github.event.inputs.branch }}" >> $GITHUB_ENV
+
+      - name: Free Disk Space (Ubuntu)
+        uses: insightsengineering/disk-space-reclaimer@v1
+        with:
+          tools-cache: true
+          android: false
+          dotnet: true
+          haskell: true
+          large-packages: true
+          swap-storage: true
+          docker-images: true
+
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "11.x"
+      - name: Configure placeholder git details
+        run: |
+          git config --global user.email "CI@cakewallet.com"
+          git config --global user.name "Cake Github Actions"
+      - name: Flutter action
+        uses: subosito/flutter-action@v1
+        with:
+          flutter-version: "3.19.6"
+          channel: stable
+
+      - name: Install package dependencies
+        run: sudo apt-get install -y curl unzip automake build-essential file pkg-config git python libtool libtinfo5 cmake clang
+
+      - name: Execute Build and Setup Commands
+        run: |
+          sudo mkdir -p /opt/android
+          sudo chown $USER /opt/android
+          cd /opt/android
+          -y curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+          cargo install cargo-ndk
+          git clone https://github.com/cake-tech/cake_wallet.git --branch ${{ env.BRANCH_NAME }}
+          cd cake_wallet/scripts/android/
```

Comment:
this should be `/linux` as well, but it seems like it doesn't matter, but it should matter cause in the Generate Externals step we don't run `./app_config.sh`